code/datums/components/material/material_container.dm

```diff
@@ -79,8 +79,9 @@
 
 	//can we insert into this container
 	if(!(mat_container_flags & MATCONTAINER_NO_INSERT))
+
 		//to insert stuff into the container
-		RegisterSignal(atom_target, COMSIG_ATOM_ITEM_INTERACTION, PROC_REF(on_item_insert))
+		RegisterSignal(atom_target, (mat_container_flags & MATCONTAINER_LOW_PRIORITY) ? COMSIG_ATOM_ATTACKBY : COMSIG_ATOM_ITEM_INTERACTION, PROC_REF(on_item_insert))
```

Comment:
I don't like this, attackby being used for interactions is intended to be deprecated, not as a way to prioritize certain interactions 

What interactions are being broken by this exactly? At a glance it seems like it might be overzealous about blocking